### PR TITLE
8303482: Update LCMS to 2.15

### PR DESCRIPTION
**Pull request by alisenchung**
Updated LCMS to 2.15.
Ran clientlibs automated tests and manually checked J2Ddemo.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8303482](https://bugs.openjdk.org/browse/JDK-8303482): Update LCMS to 2.15

### Reviewers
 * [Sergey Bylokhov](https://openjdk.org/census#serb) (@mrserb - **Reviewer**) ⚠️ Review applies to [c7aec43c](https://git.openjdk.org/jdk/pull/12967/files/c7aec43c0069dfb937adf9505e8d2c4eeca12127)
 * [Phil Race](https://openjdk.org/census#prr) (@prrace - **Reviewer**)
 * [Damon Nguyen](https://openjdk.org/census#dnguyen) (@DamonGuy - Author)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/12967/head:pull/12967` \
`$ git checkout pull/12967`

Update a local copy of the PR: \
`$ git checkout pull/12967` \
`$ git pull https://git.openjdk.org/jdk pull/12967/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 12967`

View PR using the GUI difftool: \
`$ git pr show -t 12967`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/12967.diff">https://git.openjdk.org/jdk/pull/12967.diff</a>

</details>

### CODE REVIEW

File: src/java.desktop/share/legal/lcms.md [hunk 1]
```diff
@@ -44,37 +42,37 @@ SoftSurfer makes no warranty for this code, and cannot be held
...
+<pre>
 
 ### AUTHORS File Information
 ```
 
+
 Main Author
 ------------
-Marti Maria
+Marti Maria
```
Comment (mrserb) [inline src/java.desktop/share/legal/lcms.md:54]: Please fix all these "Whitespace error"

File: src/java.desktop/share/legal/lcms.md [hunk 2]
```diff
@@ -44,12 +42,12 @@ SoftSurfer makes no warranty for this code, and cannot be held
 liable for any real or imagined damage resulting from its use.
 Users of this code must verify correctness for their application.
 
-
-</pre>
+<pre>
```
Comment (DamonGuy) [inline src/java.desktop/share/legal/lcms.md:45]: Update looks good to me. Shouldn't this stay as "/pre" though to close the opening tag on line 4?

Comment (alisenchung) [reply src/java.desktop/share/legal/lcms.md]: Just fixed

**Review by prrace — APPROVED** [review]
lgtm

**Review by DamonGuy — APPROVED** [review]

**Pull request closed without merging**